bundle/language_sw.properties

```diff
@@ -1,1480 +1,1493 @@
+angal.admission.patientfolder.dischart.btn.key                                   = S
+angal.admission.patientfolder.admchart.btn                                       = Mchoro wa kulazwa
+angal.admission.patientfolder.admchart.btn.key                                   = A
+angal.admission.patientfolder.dicom.btn                                          = DICOM
+angal.admission.patientfolder.dicom.btn.key                                      = D
+angal.admission.therapy.btn                                                      = Tiba
+angal.admission.therapy.btn.key                                                  = T
+angal.admission.merge.btn                                                        = Unganisha
+angal.admission.merge.btn.key                                                    = M
+angal.billbrowser.report.btn                                                     = Ripoti
+angal.billbrowser.report.btn.key                                                 = R
+angal.billbrowser.editbill.btn                                                   = Hariri Ankara
+angal.billbrowser.editbill.btn.key                                               = E
+angal.billbrowser.receipt.btn                                                    = Risiti
+angal.billbrowser.receipt.btn.key                                                = P
+angal.billbrowser.newbill.btn                                                    = Bili jypia
+angal.billbrowser.newbill.btn.key                                                = N
+angal.billbrowser.deletebill.btn                                                 = Futa bili
+angal.billbrowser.deletebill.btn.key                                             = D
+angal.billbrowser.today.btn                                                      = Leo
+angal.billbrowser.today.btn.key                                                  = T
+angal.groupsbrowser.groupmenu.btn                                                = GroupMenu
+angal.groupsbrowser.groupmenu.btn.key                                            = M
+angal.lab.negativeabbr.btn                                                       = N
+angal.lab.positiveabbr.btn                                                       = P
+angal.labnew.ip.btn                                                              = IP
+angal.labnew.opd.btn                                                             = OPD
+angal.newbill.findpatient.btn                                                    = Find Patient
+angal.newbill.findpatient.btn.key                                                = P
+angal.newbill.changepatient.btn                                                  = Badilisha Mgonjwa
+angal.newbill.changepatient.btn.key                                              = H
+angal.newbill.givechange.btn                                                     = Kupa chenji
+angal.newbill.givechange.btn.key                                                 = G
```

Comment:
I think the btn.key should at least present in the btn label

Reply:
Done.